lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -1062,6 +1062,92 @@ bool SessionState::deactivate_scheduled_static_rule(
   return true;
 }
 
+void SessionState::process_rule_installs(
+    std::vector<StaticRuleInstall> static_rule_installs,
+    std::vector<DynamicRuleInstall> dynamic_rule_installs,
+    RulesToProcess& to_activate, RulesToProcess& to_deactivate,
+    RulesToSchedule& to_schedule, SessionStateUpdateCriteria& session_uc) {
+  std::time_t current_time = std::time(nullptr);
+  std::string ip_addr      = config_.common_context.ue_ipv4();
```

Comment:
where is this used?

Reply:
probably not used anywhere. I was moving this function from LocalEnforcer into SessionState so still cleaning up

Reply:
sorry if I jumped the review gun